packages/table/preview/index.tsx

```diff
@@ -46,7 +46,17 @@ enum FocusStyle {
     TAB_OR_CLICK,
 };
 
+enum CellContent {
+    EMPTY,
+    CELL_NAMES,
+    LONG_TEXT,
+};
```

Comment:
logically, i'd define this immediately before `CELL_CONTENTS` instead of inserting this MutableTableState in the way.